uint64_array_test.go

```diff
@@ -0,0 +1,49 @@
+package pqextended
+
+import (
+	"database/sql"
+	"fmt"
+	"os"
+	"slices"
+	"testing"
+
+	"github.com/joho/godotenv"
+)
+
+func TestUInt64Array(t *testing.T) {
+	godotenv.Load()
+
+	postgres, err := sql.Open("postgres", os.Getenv("PQ_EXTENDED_TEST_POSTGRES_URI"))
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	_, err = postgres.Exec(`CREATE TABLE IF NOT EXISTS uint64_slices (
+		id integer,
+		uint64_slice bigint[]
+	)`)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	input := []uint64{0, 1, 2, 4294967295} // Postgres doesn't have a bigger type to support the whole uint64 range
```

Comment:
Postgres has a `numeric` type which can be used here. It's slower than the other number types, but it allows for much larger numbers (131,072 digits before a decimal, and 16,383 after). Postgres allows the exact precision to be specified by the user as well so it can be clamped to just the uint64 range